discord/role.py

```diff
@@ -68,22 +68,37 @@ class RoleTags:
         The bot's user ID that manages this role.
     integration_id: Optional[:class:`int`]
         The integration ID that manages the role.
+    subscription_listing_id: Optional[:class:`int`]
+        The subscription SKU and listing ID of the role.
+
+        .. versionadded:: 2.7
     """
 
     __slots__ = (
         "bot_id",
         "integration_id",
+        "subscription_listing_id",
         "_premium_subscriber",
+        "_available_for_purchase",
+        "_guild_connections",
     )
 
     def __init__(self, data: RoleTagPayload):
         self.bot_id: int | None = _get_as_snowflake(data, "bot_id")
         self.integration_id: int | None = _get_as_snowflake(data, "integration_id")
-        # NOTE: The API returns "null" for this if it's valid, which corresponds to None.
+        self.subscription_listing_id: int | None = _get_as_snowflake(
+            data, "subscription_listing_id"
+        )
+        # NOTE: The API returns "null" for this if the following tags True, and doesn't return them at all if False.
```

Comment:
See suggestion below + state whether it's "if all are True" or "if any are True".
```suggestion
        # NOTE: The API returns "null" for this if the following tags are True,
        # otherwise it doesn't return them at all.
```

Reply:
Changed it pls see if it's good like now

Reply:
That's pretty clear, thanks.